compiler-rt/lib/sanitizer_common/sanitizer_linux.cpp

```diff
@@ -2154,7 +2154,8 @@ bool SignalContext::IsTrueFaultingAddress() const {
 UNUSED
 static const char *RegNumToRegName(int reg) {
   switch (reg) {
-#  if SANITIZER_LINUX && SANITIZER_GLIBC || SANITIZER_NETBSD
+#  if (SANITIZER_LINUX && SANITIZER_GLIBC && !SANITIZER_ANDROID) || \
```

Comment:
I m surprised you need to indicate android since there is glibc involved but lgtm..

Reply:
I see a lot of places inside compiler_rt where we have `SANITIZER_LINUX && !SANITIZER_ANDROID`

Reply:
i don't know actually if android uses glibc but it seems that yes

Reply:
I hope this is the last patch with adjustments of preprocessor directives

Reply:
hmmm strange still because android libc is bionic :)

Reply:
Of course this is strange. __GLIBC__ is defined somewhere

Reply:
> Of course this is strange. **GLIBC** is defined somewhere

See sanitizer_platform.h comment line 21. Anyhow, feel free to merge or wait @vitalybuka eventually.

Reply:
Maybe, but i cannot quicky deploy android env and check this precisely...
So i think we can leave this patch unchanged to make buildbots green

Reply:
Lets wait @vitalybuka

Reply:
@vitalybuka tell us please what is the correct combination of macroses here :)

Reply:
Android does not use SANITIZER_GLIBC

Reply:
On revision from the build error, there is no
`if SANITIZER_LINUX && SANITIZER_GLIBC || SANITIZER_NETBSD` in `RegNumToRegName`